src/test/java/pages/SwagLabsLoginPage.java

```diff
@@ -0,0 +1,39 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SwagLabsLoginPage {
+
+    @FindBy(id ="user-name") WebElement inputUserField;
+    @FindBy(id ="password") WebElement inputPassField;
+    @FindBy(id ="login-button") WebElement loginBtn;
+
+    WebDriver driver;
+    WebDriverWait wdWait;
+
+    public SwagLabsLoginPage(WebDriver driver, WebDriverWait wdWait)
+    {
```

Comment:
The java convention is to have the opening braces on the same line as the parameter declaration